app/code/community/Ebanx/Gateway/Block/Health/Check.php

```diff
@@ -35,9 +28,11 @@ public function setConfigData()
     }
 
     /**
+     * @param Varien_Data_Form_Element_Abstract $element
+     *
      * @return void|string
      */
-    public function render()
+    public function render(Varien_Data_Form_Element_Abstract $element)
```

Comment:
How our e2e not get this error ?

Reply:
This error did not occur in our docker environment, in which our e2e runs